### PR TITLE
fix/BODS-2634: FlexibleServiceTimes - PTI Validation rule fix

### PR DESCRIPTION
**Pull request by ndemenezes**

### CODE REVIEW

File: transit_odp/data_quality/pti/functions.py [hunk 1]
```diff
@@ -408,3 +407,23 @@ def get_stop_point_ref_list(stop_points, ns):
...
+    flexible_vehiclejourneys = vehiclejourneys[0].xpath(
+        "x:FlexibleVehicleJourney", namespaces=ns
+    )
+    if flexible_vehiclejourneys:
+        for flexible_journey in flexible_vehiclejourneys:
+            flexible_service_times = flexible_journey.xpath(
+                "x:FlexibleServiceTimes", namespaces=ns
+            )
+            if len(flexible_service_times) == 0:
+                return False
```
Comment (amitkpmguk) [inline transit_odp/data_quality/pti/functions.py:427]: In the task description following validation is mentioned
"FlexibleServiceTimes must be populated as a valid time"

But we are only checking for existence of the time element,

Comment (ndemenezes) [reply transit_odp/data_quality/pti/functions.py:427]: @amitkpmguk This has been confirmed that the schema check covers this

**Review by amitkpmguk — DISMISSED** [review]
Looks good to me 👍

**Review by amitkpmguk — APPROVED** [review]
Looks good to me 👍

**Pull request merged**